lang/java/grpc/pom.xml

```diff
@@ -21,8 +21,8 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.apache.avro</groupId>
     <artifactId>avro-parent</artifactId>
+    <groupId>org.apache.avro</groupId>
     <version>1.9.0-SNAPSHOT</version>
```

Comment:
The convention is to have `<groupId>` before `<artififactId>`. Has some tool changed the order here accidentally?

Reply:
Hi, actually I noticed that this was in this order in other pom files so I changed to be consistent but I can revert it if you prefer.